ts_discomfort_index.json

```diff
@@ -0,0 +1,41 @@
+{
+  "information": {
+    "summary": "This file contains a series of validation tables for discomfort_index() in pythermalcomfort, jsthermalcomfort, calcDisc() in R",
+    "version": "1.0.0",
+    "license": "MIT"
+  },
+  "tolerance": {},

```

Comment:
missing

Reply:
Hi Dr. Federico, I noticed that the tolerance value for discomfort_index is currently missing[link](https://github.com/FedericoTartarini/validation-data-comfort-models/issues/1). Would you be able to provide us with a recommended tolerance value for that?

Reply:
0.1 I also updated the issue.

Reply:
@number9527-12 the tolerance value is still missing